SplunkRumWorkspace/SplunkRum/SplunkRum/NotificationEvents.swift

```diff
@@ -0,0 +1,47 @@
+//
+/*
+Copyright 2021 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import Foundation
+import UserNotifications
+import UIKit
+
+class NotificationEvents {
+
+    @objc func receiveRemoteNotification(_ application: UIApplication, didReceiveRemoteNotification userInfo: [AnyHashable: Any], fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
+
+        }
+
+    private func swizzleDidReceiveRemoteNotification() {
```

Comment:
Rather than swizzling is there a solution here that involve listening to `NotificationCenter` events?  This has been used heavily elsewhere in this repo.  If you have a toy app that can receive remote events then you can simply listen to all names and see if this approach would work.

Reply:
In the userNotificationCenter didReceive delegate method, we receive an event regarding the notification tab. I create a function to get UNNotificationResponse and I call it from the end-user app's UNUserNotificationCenterDelegate method.

@objc public class func notificationTab(_ userInfo: UNNotificationResponse) {
        userNotificationCenterTap(userInfo: userInfo)
    }

Please let me know your opinion.

Reply:
Here we are using the notificationTab argument to capture the notification click event. I update the PR. 
Please let me know your opinion.

Reply:
Here we are using the notificationTab argument to capture the notification click event. I update the PR.
Please let me know your opinion.